packages/devtools_app/lib/src/ui/flutter/flutter_icon_renderer.dart

```diff
@@ -99,7 +99,14 @@ class _ColorIconPainter extends CustomPainter {
 }
 
 Widget _computeIconWidget(DevToolsIcon icon) {
-  if (icon is UrlIcon) {
+  if (icon is OcticonIcon) {
```

Comment:
This switch-on-type pattern is likely better solved with polymorphism.

Is it possible to add a Widget build() method to the DevToolsIcon class?

Reply:
we can't easily use polymorphism because of 
html_icon_renderer.dart
once we remove the html version I expect this code will be deleted and we will just make the DevToolsIcon classes BE widgets with the `build()` method on DevToolsIcon providing the polymorphism.